netlink-packet-wireguard/src/nlas/allowedip.rs

```diff
@@ -0,0 +1,80 @@
+use crate::{constants::*, raw::*};
```

Comment:
I'd prefer not to have wildcard imports for modules other than `constants`. I don't always have rust-analyzer enabled, and when I don't I like being able to quickly see from where a given symbol is being imported.